packages/formatron-react/src/form/fluentui/Number.tsx

```diff
@@ -0,0 +1,43 @@
+import {ITextFieldProps, TextField} from '@fluentui/react/lib/TextField';
+import {NumberType} from '@idelic/formatron/lib/dataTypes/number';
+import React from 'react';
+
+import {FieldObjectDataType} from '../Form';
+import {CommonFieldProps} from '../types';
+
+export interface NumberProps extends CommonFieldProps {
+  field: FieldObjectDataType<NumberType>;
+  textFieldProps?: ITextFieldProps;
+}
+
+export const Number: React.FC<NumberProps> = ({
+  field,
+  textFieldProps = {},
+  disabled = false,
+  label
+}) => {
+  const {
+    value,
+    setValue,
+    error,
+    dataType,
+    setTouched,
+    isTouched,
+    isRequired,
+    isSubmitting
+  } = field;
+
+  return (
+    <TextField
+      value={value ?? ''}
+      type="number"
```

Comment:
Should this be single quotes?

Reply:
`.prettierrc` is missing the `jsxSingleQuote` rule. We can add it as part of this PR if you think it's worth it.

Reply:
eh. I wouldn't worry about it, I was just verifying.

Reply:
We could add it for consistency if you want, but it's not a showstopper or anything. If we do plan on adding it eventually, then we might as well do it now.